download/releaseNotes/releaseNotes7.adoc

```diff
@@ -5,6 +5,12 @@
 :awestruct-release_notes_version: 7
 :awestruct-release_notes_version_qualifier: Final
 
+== New and noteworthy: Engine 7.33.0.Final
+
+=== Conditional propagation in constraint streams
+
+The constraint streams API continued to evolve, gaining support for https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreamsConditionalPropagation[conditional propagation].
```

Comment:
This is a broken link, because latestFinal is still 7.33.

Normally, in N&N we show a summary immediately, without the users having to click through.
So instead of having a link here, I 'd just copy paste a small ifExists example source code. No link needed even, if they want more information, that's what the manual is for.

Reply:
Understood. That said, this really shouldn't be published before 7.33 is out, right?
Because otherwise the feature itself isn't really available.

Reply:
It's clearly noted that this is for 7.33. Merging it is the lesser evil: not merging it causes merge conflicts when we add other stuff, or forgetting to announce at all. So historically, we merge it.